packages/devtools_app/lib/src/shared/table.dart

```diff
@@ -869,7 +865,7 @@ class _TableState<T> extends State<_Table<T>> with AutoDisposeMixin {
   }
 
   void _onActiveSearchChange() async {
-    final activeSearch = widget.activeSearchMatchNotifier.value;
+    final T? activeSearch = widget.activeSearchMatchNotifier!.value;
```

Comment:
we shouldn't need type on LHS. type should be inferred from RHS

Reply:
done